src/app/devices/device-detail/device-detail.component.scss

```diff
@@ -37,4 +37,24 @@ p{
   span{
     color:black
   }
+}
+
+.oobpowerActions {
```

Comment:
ideally we aren't adding custom css if we can avoid it. I think we can accomplish this by using the the flex tags provided with angular flex. Additionally, these aren't wrapping like they should when the window is smaller.
![image](https://user-images.githubusercontent.com/22204562/170548848-60204aae-e78e-4bf5-bb9e-adcae16d647f.png)